src/app/pages/fields/subcategories/subcategories.component.ts

```diff
@@ -25,7 +25,7 @@ export class SubcategoriesComponent implements OnInit {
   }
 
   onNavigationChange(event): void{
-
+    console.log(event);
```

Comment:
The event handler is useless because the component renders link. You should also update the template.